text/0000-document_all_features.md

```diff
@@ -0,0 +1,275 @@
+    2. We could automatically render the text of the RFCs into part of the documentation used on the site (via submodules and the existing tooling around Markdown documents used for Rust documentation).
+
+    However, for all the reasons highlighted above in [**Motivation: The Current Situation**][current-situation], RFCs and their associated threads are *not* a good canonical source of information on language features.
+
+- **Add a rule for the standard library but not for language features.**
+
+    This would basically just turn the _status quo_ into an official policy. It has all the same drawbacks as no change at all, but with the possible benefit of enabling automated checks on standard library documentation.
+
+- **Add a rule for language features but not for the standard library.**
+
+    The standard library is in much better shape, in no small part because of the ease of writing inline documentation for new modules. Adding a formal rule may not be necessary if good habits are already in place.
+
+    On the other hand, having a formal policy would not seem to *hurt* anything here; it would simply formalize what is already happening (and perhaps, via linting attributes, make it easy to spot when it has failed).
+
+- **Eliminate the reference entirely.**
+
+    Since the reference is already substantially out of date, it might make sense to stop presenting it publicly at all, at least until such a time as it has been completely reworked and updated.
+
+    The main upside to this is the reality that an outdated and inaccurate reference may be worse than no reference at all, as it may substantially mislead Rust users.
+
+    The main downside, of course, is that this would leave very large swaths of the language basically without *any* documentation, and even more of it only documented in RFCs than is the case today.
+
+
+[RFC 1270]: https://github.com/rust-lang/rfcs/pull/1270
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+- How do we clearly distinguish between features on nightly, beta, and stable Rust—in the reference especially, but also in the book?
+- How will the requirement for documentation in the reference be enforced?
+- Given that the reference is out of date, does it need to be brought up to date before beginning enforcement of this policy?
+- Given that the book is in the process of a rewrite for print publication, how shall we apply this requirement to RFCs merged during its development?
+- For the standard library, once it migrates to a crates structure, should it simply include the `#[forbid(missing_docs)]` attribute on all crates to set this as a build error?
+- Is a documentation subteam, _a la_ the one used by Ember, worth creating?
```

Comment:
I believe this question can be removed from the unresolved questions list since [we have a documentation subteam now](https://github.com/rust-lang/rfcs/blob/master/text/1683-docs-team.md) :)